### ISSUE TITLE
AMA Intake | UX Priorities

### ISSUE THREAD

**Issue by carodew** [department-of-veterans-affairs/caseflow#13840]
@carolaponce commented on [Fri Oct 19 2018](https://github.com/department-of-veterans-affairs/appeals-team/issues/1033)

>Taking Abby's issue and continue to use this list as a source of UX deliverables + priorities.
#833 

# **Working **On****

### Initiative: Supporting intake and disposition tracking for non-compensation business lines
>Non-compensation functionality, which is basically a modified version of our compensation features. The main pieces are intake, dispositions, and possibly a generic queue/work distribution mechanism. Next steps: refine draft designs. Initial Demo done - will collect feedback from all teams and push screens to production (Date end of October?)

Process documentation #1028 

- [x] Work on rough draft mocks for development - #893, #906 #920  
- [x] User testing future state with non-compensation lines of business - #893, #906  
- [x] Figure out work distribution/queue reusability
- [ ] Any adjustments to opt-in functionality for non-comp (TBD)
- [x] Any adjustments to issue validations/alerting for non-comp (TBD)

### Initiative: Alerting users about invalid AMA issues
>Flag ineligible/untimely/duplicate issues to users, so they can either override the check and process the form anyway, or notify the Veteran that they submitted a form with ineligible issues. | 
Suzanne finished these screens.
I need to figure out if any of these screens need modifications for Non-comp.

- [x] Duplicate of an issue already in an AMA lane - [#5228](https://github.com/department-of-veterans-affairs/caseflow/issues/5228)
- [x] Issue is older than 1 year - #915 
- [x] HLR-> HLR [#5422](https://github.com/department-of-veterans-affairs/caseflow/issues/5244)
- [x] For non-comp users: https://github.com/department-of-veterans-affairs/appeals-design-research/issues/1000

# **Next **Priorities****

### Initiative: User Research Plan for December Pilot 
 Can we schedule this in Oakland? @shanear?
What to plan for (WIP)
- [ ] Users
- [ ] Priorities/hypotheses
- [ ] Scripts
- [ ] Contextual Inquiries
- [ ] Observants/Volunteers to take notes
- [ ] Testing software/technology

### Initiative: Tracking power(s) of attorney at the issue level
> Veterans will be able to appoint a per-issue representative, using the 21-22a form, so we want to create an intake path for this form. This functionality is somewhat standalone, and will likely involve close collaboration with Queue designers. Timing: sometime prior to February.  For VBA, they'll view the POA in VBMS; for the Board, they'll view it in Queue.

- [ ] 21-22A [#6206](https://github.com/department-of-veterans-affairs/caseflow/issues/6206)
  - Including splitting appeals as necessary?
  - Waiting on VBMS dev work to create these at EP level
  - Board and VBA

### Initiative: searching/viewing status of SCs/HLRs/Appeals
> Queue will add information about HLRs/SCs to Queue - still needs definition (Shane has some thoughts on what might go here, I think the plan is for Queue to develop it, but we need to know what they need)

- [x] Ability to search for an Intake - (Queue search) - [#6710](https://github.com/department-of-veterans-affairs/caseflow/issues/6710)
- [ ] Displaying POA info to AMO/supervisors (Queue search)

### Initiative: Establishing AMA reviews 
> This is the "core path" for establishing AMA appeals. Most of the backbones of this work is complete, but there are some nitty-gritty details/scenarios that still require design work. Carola will own setting up any future user testing sessions. The bulk of this work should be ready for testing during the pilot (likely ~December 2018). 

- [x] July pilot: User testing BEAAM (Board’s Early Applicability of Appeals
Modernization) appeals with Board Intake team - delayed again until the week of July 23 #929 
- [x] BEAAM revisions/usability fixes - focused on issue selection improvements
- [x] Handling confirmations - "partially" confirmed EP creation 
- [ ] Pre-pilot user and stakeholder feedback
- [ ] Early Applicability pilot (likely December) user shadowing - VBA and Board

### Initiative: Editing reviews post-intake
> Users can add or remove SC/HLR issues after intake, via a link in VBMS to a page in Caseflow. For appeals, we'll need to collaborate with the Queue team, who will own the flow for attorneys to edit issues in Queue after intake. This is mostly finished on the VBA side; keep an eye on opportunities to collaborate with Queue. This is currently in development (almost finished, as of Oct.)

- [x] Modifying a review after intake (edit) - VBA focus for now - #862
- [x] Syncing with Queue designers on post-intake issue edits from Queue. [See Mural](https://app.mural.co/t/workqueue2001/m/workqueue2001/1530197104426/ad25f0911c9e16551dbef41615fbd70c96c91f27)
- [x] Automatic rollbacks when an EP is canceled

### Initiative: supporting opt-ins from the legacy system to AMA
> A Veteran can opt their legacy appeal into AMA after receiving an SOC/SSOC decision from VBA. We'll need a way for them to identify opt-ins and close out VACOLS issues.

- [x] Legacy opt-in handling - #919, [Mural](https://app.mural.co/t/workqueue2001/m/workqueue2001/1537976997697/bf0581faa1dc3e12543d9c02d1fd78a37264da3e)

## Parking lot
- [x] Redesigning the issue selection/escalation flow
- [x] Testing new issue selection/escalation flow
- [x] What should happen with unknown issues? How are comments displayed?
- [x] Claimants/payees other than Veteran - our MVP handling is to show the entire list of all available options, but there may be smarter ways to limit/predict the list of possible payees, based on the claimant and/or Veteran. [#6842](https://github.com/department-of-veterans-affairs/caseflow/issues/6842)
- [ ] Cancellation modal -  currently, the cancelation reasons are based on RAMP. We should revisit to make sure they apply to AMA
- [ ] Manager page - there's an existing RAMP manager page that shows canceled EPs. What, if anything, should happen to this for (or after) AMA?
- [ ] Investigate auto-generating letters to notify Veterans of receipt/action on their request - #703 
- [ ] Work history for claims assistants
- [ ] Manager tools (see rough sketch) - #859 (note: this may not be necessary if we're able to give supervisor users access to pieces of Queue)
- [ ] Ability to "undo" an intake? (supervisors)

---

@carolaponce commented on [Wed Oct 24 2018](https://github.com/department-of-veterans-affairs/appeals-team/issues/1033#issuecomment-432721229)

From @shanear 

department-of-veterans-affairs/caseflow#7383
department-of-veterans-affairs/caseflow#5695
department-of-veterans-affairs/caseflow#7515
#1023

---

@jimruggiero commented on [Sat Sep 07 2019](https://github.com/department-of-veterans-affairs/appeals-team/issues/1033#issuecomment-529157625)

@leikkisa Putting this list on your radar for Foxtrot roadmap prep I'm going to start when I return from PTO. Prep work will be to organize all that's currently in the backlog, for Foxtrot and then meet with you to review and get your input on tech debt/prioritization and finesse my prep docs. I'd like your thoughts on the unchecked items above: is the list still accurate? any of this roll into tech debt/improvements? Not an immediate priority, but wanted you to have a few weeks to think on this. I expect to have my stuff ready for us to discuss between 9/25-10/1.

---

@jimruggiero commented on [Sat Sep 07 2019](https://github.com/department-of-veterans-affairs/appeals-team/issues/1033#issuecomment-529157705)

@carodew mind if I put this in your backlog until we determine next steps for this outstanding work?

---

@carodew commented on [Mon Sep 09 2019](https://github.com/department-of-veterans-affairs/appeals-team/issues/1033#issuecomment-529517580)

@jimruggiero sure, it can hangout with us designers. This seems like something you and I should talk through to figure out what the intent is, current needs, and break it down into smaller chunks? 

---

@jimruggiero commented on [Mon Sep 09 2019](https://github.com/department-of-veterans-affairs/appeals-team/issues/1033#issuecomment-529537561)

@carodew Agreed. I'm thinking we can chat late next week or the week after. There's no rush on this. 

---

@carodew commented on [Tue Mar 31 2020](https://github.com/department-of-veterans-affairs/appeals-team/issues/1033#issuecomment-606787223)

@jimruggiero should we bounce this back to @msj0nes and @rutvigupta-design to determine relevancy/next steps? This seems like a pre-epic or pre-product brief set of notes, but if it's not something to prioritize in the short term I think labeling and closing is probably the best approach. 

---

@rutvigupta-design commented on [Tue Mar 31 2020](https://github.com/department-of-veterans-affairs/appeals-team/issues/1033#issuecomment-606788443)

@carodew @jimruggiero I would love to work with @msj0nes to break this into smaller incremental improvement tickets.

**Comment by msj0nes** [comment]
I believe this should be closed. Design work for foxtrot will be via more current stakeholder communications.

**Comment by msj0nes** [comment]
Closing. This isn't work currently prioritized . Will depend on more recent stakeholder feedback to craft UX priorities.
